Apps/WebClient/src/NewClientApp/src/components/private/profile/UserProfileAddressComponent.vue

```diff
@@ -0,0 +1,135 @@
+        return !hasAddress.value;
+    }
+
+    const arrayEqual = (a: string[], b: string[]) =>
+        a.length === b.length && a.every((v, i) => v === b[i]);
+
+    const streetLinesMatch = arrayEqual(
+        postalAddress.value.streetLines,
+        physicalAddress.value.streetLines
+    );
+    const cityMatches =
+        postalAddress.value.city === physicalAddress.value?.city;
+    const stateMatches =
+        postalAddress.value.state === physicalAddress.value?.state;
+    const postalCodeMatches =
+        postalAddress.value.postalCode === physicalAddress.value?.postalCode;
+
+    return streetLinesMatch && cityMatches && stateMatches && postalCodeMatches;
+});
+const postalAddressLabel = computed(() =>
+    !isSameAddress.value || (physicalAddress.value && !postalAddress.value)
+        ? "Mailing Address"
+        : "Address"
+);
+</script>
+
+<template>
+    <div class="mb-4">
+        <SectionHeaderComponent
+            data-testid="postal-address-label"
+            :title="postalAddressLabel"
+        />
+        <div
+            v-if="postalAddress"
+            data-testid="postal-address-div"
+            class="text-body-1"
+        >
+            <div
+                v-for="(item, index) in postalAddress.streetLines"
+                :key="index"
+            >
+                {{ item }}
+            </div>
+            <div>
+                {{ postalAddress.city }}, {{ postalAddress.state }},
+                {{ postalAddress.postalCode }}
+            </div>
+        </div>
+        <div
+            v-else
+            data-testid="no-postal-address-text"
+            class="text-body-1 font-italic"
+        >
+            No address on record
+        </div>
+    </div>
+    <div
+        v-if="!isSameAddress"
+        data-testid="physical-address-section"
+        class="mb-4"
+    >
+        <SectionHeaderComponent
+            data-testid="physical-address-label"
+            title="Physical Address"
+        />
+        <div
+            v-if="physicalAddress"
+            data-testid="physical-address-div"
+            class="text-body-1"
+        >
+            <div
+                v-for="(item, index) in physicalAddress.streetLines"
+                :key="index"
+            >
+                {{ item }}
+            </div>
+            <div>
+                {{ physicalAddress.city }}, {{ physicalAddress.state }},
+                {{ physicalAddress.postalCode }}
+            </div>
+        </div>
+        <div
+            v-else
+            data-testid="no-physical-address-text"
+            class="text-body-1 font-italic"
+        >
+            No address on record
+        </div>
+    </div>
+    <div v-if="hasAddress && isSameAddress" class="mb-4 text-body-1">
+        If this address is incorrect, update it
+        <a
+            href="https://www.addresschange.gov.bc.ca/"
+            target="_blank"
+            rel="noopener"
+            >here</a
+        >
+        <span>.</span>
```

Comment:
Surely <span>necesssary?, is it there just so we can see it?

same below

Reply:
No, it's not necessary, so long as the period is moved directly after the ending `</a>` rather than on a new line.  Will fix.